docs/api/sql/Constructor.md

```diff
@@ -146,14 +204,20 @@ Format:
 Since: `v1.0.0`
 
 Spark SQL example:
+
 ```sql
-SELECT ST_GeomFromWKB(polygontable._c0) AS polygonshape
-FROM polygontable
+SELECT ST_GeomFromWKB([01 02 00 00 00 02 00 00 00 00 00 00 00 84 D6 00 C0 00 00 00 00 80 B5 D6 BF 00 00 00 60 E1 EF F7 BF 00 00 00 80 07 5D E5 BF])
```

Comment:
Please specify this supports EWKB, and put an example for it: 01010000a0e6100000000000000000f03f000000000000f03f000000000000f03f